### PR TITLE
Added Node.js binding

### PR DESCRIPTION
**Pull request by XadillaX**

### CODE REVIEW

File: download/bindings-fr.php [hunk 1]
```diff
@@ -178,6 +178,12 @@
             <td>2.5.1</td>
             <td>Guillaume Derex</td>
         </tr>
+        <tr>
+            <th rowspan="1">Node.js</th>
+            <td><a href="https://github.com/XadillaX/node-sfml" class="website">sfml.js</a></td>
```
Comment (eXpl0it3r) [inline download/bindings-fr.php:183]: Wouldn't node-sfml make more sense instead of sfml.js given that the project is called node-sfml?

Comment (XadillaX) [reply download/bindings-fr.php:183]: It's package name in npm is [sfml.js](https://www.npmjs.com/package/sfml.js).

**Comment by eXpl0it3r** [comment]
Thanks for providing this PR, I'll update the website soon

**Comment by XadillaX** [comment]
Hi, I'm here just to ask when will the website update :)

**Comment by eXpl0it3r** [comment]
I'm very sorry, I thought that I updated it already. Done so now.

**Pull request merged**